### PR TITLE
add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by EvgeniiaVak**
refs #9

### CODE REVIEW

**Comment by EvgeniiaVak** [comment]
What else do we need to add here?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Contributions are welcome! Check out [good first issues](https://github.com/DeXter-on-Radix/website/issues?labels=good+first+issue&sort=updated&state=open) or create a new one to start a discussion.
+
+Here are a few guidelines to follow:
+
+- **[Use Prittier](https://prettier.io/)** and automatic code formatting to keep PRs focused on the changes
```
Comment (fliebenberg) [inline CONTRIBUTING.md:5]: Should be [Use Prettier]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Contributions are welcome! Check out [good first issues](https://github.com/DeXter-on-Radix/website/issues?labels=good+first+issue&sort=updated&state=open) or create a new one to start a discussion.
```
Comment (fliebenberg) [inline CONTRIBUTING.md:1]: maybe change to "... or create a new issue to start a discussion."

**Review by fliebenberg — COMMENTED** [review]
Few minor changes suggested.

**Comment by SmashingBumpkin** [comment]
A few things that might be worth adding:

Links to the tg and/or discord
Naming and formatting conventions, particularly capitalization guidelines
A note about not commiting directly to main (maybe goes without saying but since we're on a free repo it is possible!)

**Comment by EvgeniiaVak** [comment]
> not commiting directly to main
yeah, I'll add this, thanks, but I think it's not possible, at least I tried it setup this way - you shouldn't be able to push commits without a PR directly to main, if you find any option on how to do that - please flag that, we'll try to fix it

**Comment by EvgeniiaVak** [comment]
> capitalization guidelines
like what?

**Comment by SmashingBumpkin** [comment]
So I'm pretty new to js/tsx, but (my?) convention in Java is classes are capitalized CamelCase, constants are all caps, methods are camelCase without caps, etc etc.On Jul 5, 2023 14:10, EvgeniiaVak ***@***.***> wrote:

capitalization guidelines
like what?

**Comment by EvgeniiaVak** [comment]
@SmashingBumpkin I added the settings to eslint so that it would throw an error if new MyClass() is not Capitalized, and if myFunction or myProperty are not camelCase.

**Review by SmashingBumpkin — APPROVED** [review]
Looks great. ESLint is new to me but looks really useful.

**Pull request merged**